drizzle-orm/src/sqlite-proxy/session.ts

```diff
@@ -131,10 +137,15 @@ export class PreparedQuery<T extends PreparedQueryConfig = PreparedQueryConfig>
 
 		const clientResult = await this.client(query.sql, params, 'get');
 
+		if (Array.isArray(clientResult.rows) && clientResult.rows.length === 0) {
+			return undefined;
+		}
```

Comment:
According to the [docs](https://orm.drizzle.team/docs/get-started-sqlite#http-proxy) and the reference implementation, when no row is fetched, the output by the proxy server is an empty string. When being deserialized, this throws an error, returning `{ rows: [] }`. Instead of checking `….rows === undefined`, I therefore checked for an array of length 0.